openassessment/templates/openassessmentblock/student_training/student_training_closed.html

```diff
@@ -25,9 +24,9 @@
         <div class="wrapper--step__content">
 
             <div class="step__message message message--incomplete">
-                <h3 class="message__title">{% trans "The Due Date for This Step Has Passed" %}</h3>
+                <h5 class="message__title">{% trans "Status" %}</h5>
                 <div class="message__content">
-                    <p>{% trans "This step is now closed.You can no longer continue with this assignment, and you'll receive a grade of Incomplete." %}</p>
+                    <p>{% trans "The Due Date for This Step Has Passed. This step is now closed. You can no longer continue with this assignment, and you'll receive a grade of Incomplete." %}</p>
```

Comment:
Sentence case for "The due date for this step has passed." 
Remove "This step is now closed."
Replace "you'll" with "you will"